test/transaction_module.test.ts

```diff
@@ -33,6 +34,50 @@ const TMABI = [
     "function executeTransaction(address)",
     "function executeLargeTransaction(address, address, uint, bytes)"
 ]
+const ERC20TransferABI = [
```

Comment:
Get the artifacts from artifacts according to https://hardhat.org/guides/compile-contracts.html#reading-artifacts.